app/Providers/AppServiceProvider.php

```diff
@@ -22,7 +24,8 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register(): void
     {
-        //
+        $this->app->singleton('optimize', fn(): Optimize => new Optimize());
+        $this->app->singleton('delivery', fn(): Delivery => new Delivery());
```

Comment:
If you want to keep the return type hint

```suggestion
        $this->app->singleton('optimize', fn(): Optimize => app()->make(Optimize::class);
        $this->app->singleton('delivery', fn(): Delivery => app()->make(Delivery::class);
```

Reply:
I used the other suggestion.